ariadne/cardano/src/Ariadne/MainTemplate.hs

```diff
@@ -116,7 +117,8 @@ initializeEverything MainSettings {..}
   let
     mkWalletFace :: (Doc -> IO ()) -> WalletFace
     walletInitAction :: IO ()
-    (mkWalletFace, walletInitAction) =
+    walletCheckKeys :: IO ()
+    (mkWalletFace, walletInitAction, walletCheckKeys) =
```

Comment:
Why not make `walletCheckKeys` part of `walletInitAction`?

Reply:
It definitely would be better, i just thought that i shoud not run async computation in walletInitAction. Now i can't say why. 
Should i move it ? It is easy.

Reply:
Yes, I also can't say why you shouldn't.